sql-private-pool/cloudbuild.yaml

```diff
@@ -38,15 +38,17 @@ steps:
 options:
   pool:
     name: projects/$PROJECT_ID/locations/us-central1/workerPools/private-pool
-  dynamic_substitutions: true
+  dynamicSubstitutions: true
 
 substitutions:
   _DATABASE_PASSWORD_KEY: database_password
   _DATABASE_TYPE: postgres
-  _IMAGE_NAME: gcr.io/${PROJECT_ID}/sample-private-pool
+  _AR_REPO_REGION: us-central1
```

Comment:
question: Is `_AR_REPO` a convention we're using? Thinking about this from a Cloud Build docs perspective I wonder if it might be odd.

Reply:
It was confused in an earlier PR with "REPO_NAME", which is a built-in subvar